<ui version="4.0" >
 <class>TableWidgetNoColTestFrameClass</class>
 <widget class="QFrame" name="TableWidgetNoColTestFrameClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>TableWidgetNoColTestFrame</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>0</y>
     <width>711</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title" >
    <string>TableWidget (no clumn names)</string>
   </property>
   <widget class="QPushButton" name="pushButton" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>170</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>select</string>
    </property>
   </widget>
   <widget class="BasarTableWidget" name="tableWidget" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>671</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>460</y>
     <width>711</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title" >
    <string>TableWidget (normal)</string>
   </property>
   <widget class="QPushButton" name="pushButton_3" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>170</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>select</string>
    </property>
   </widget>
   <widget class="BasarTableWidget" name="tableWidget_3" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>671</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>230</y>
     <width>711</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title" >
    <string>empty TableWidget</string>
   </property>
   <widget class="QPushButton" name="pushButton_2" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>170</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>select</string>
    </property>
   </widget>
   <widget class="BasarTableWidget" name="tableWidget_2" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>671</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_All" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>700</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>select All</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>BasarTableWidget</class>
   <extends>QTableWidget</extends>
   <header>libbasarqtwidget_tablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
